<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormGestaoEstoque</class>
 <widget class="QDialog" name="FormGestaoEstoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="statusbarLayout"/>
   </item>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabEstoque">
     <property name="styleSheet">
      <string notr="true">QLineEdit, QSpinBox, QDoubleSpinBox{
	border: 1px solid #bfbebd;
	border-radius: 10px;
	padding: 2px;
}

QSpinBox::up-button, QDoubleSpinBox::up-button  {
	subcontrol-origin: margin;
	subcontrol-position: top right;
	right: 5px;
}
QSpinBox::down-button, QDoubleSpinBox::down-button {
	subcontrol-origin: border;
	subcontrol-position: bottom right;
	right: 5px;
}</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab1">
      <attribute name="title">
       <string>Gestão de produtos</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTableWidget" name="tabelaProdutos"/>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="gestaoFuncLayout" columnstretch="1,3,0,2">
         <item row="1" column="1">
          <widget class="QPushButton" name="filtrarButton">
           <property name="toolTip">
            <string>Filtrar</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
padding: 5px;
border: none;
border-radius: 10px;
background-color: #6d747c;
color: #fff;
}

:hover {
background-color: #7c8289;
}</string>
           </property>
           <property name="text">
            <string>Filtrar</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="editarButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="toolTip">
            <string>Editar</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	padding: 5px;
	background-color: #ffffff; 
	color: #000000;
	border: 1px solid #fdc005;
	border-radius: 10px;
} 

:hover {
	background-color: #fdc005;
}</string>
           </property>
           <property name="text">
            <string> Editar</string>
           </property>
           <property name="icon">
            <iconset resource="Recursos.qrc">
             <normaloff>:/imgs/imagens/editar.png</normaloff>:/imgs/imagens/editar.png</iconset>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPushButton" name="deletarButton">
           <property name="toolTip">
            <string>Deletar</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	padding: 5px;
	border: 1px solid #dc3643;
	border-radius: 10px;
	background-color: #fff;
	color:#dc3643;
}

:hover {
	background-color: #dc3643;
	color: #fff;
}</string>
           </property>
           <property name="text">
            <string> Deletar</string>
           </property>
           <property name="icon">
            <iconset resource="Recursos.qrc">
             <normaloff>:/imgs/imagens/deletar.png</normaloff>:/imgs/imagens/deletar.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="0" rowspan="2">
          <widget class="QGroupBox" name="groupBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
           <property name="title">
            <string>Filtrar por:</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QRadioButton" name="codigoRadioButton">
              <property name="text">
               <string>Código</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="descRadioButton">
              <property name="text">
               <string>Descrição</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="busca">
           <property name="placeholderText">
            <string>Código</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="2">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab2">
      <attribute name="title">
       <string>Adicionar produto</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="5" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="criarButton">
           <property name="toolTip">
            <string>Adicionar produto</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
margin-top: 25px;
margin-bottom: 10px;
padding: 5px;
border-radius: 10px;
background-color: #26a844;
color: #fff;
}

QPushButton:hover {
background-color: #22973d;
}</string>
           </property>
           <property name="text">
            <string>Criar novo produto</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="limparButton">
           <property name="toolTip">
            <string>Limpar campos</string>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
margin-top: 25px;
margin-bottom: 10px;
padding: 5px;
border: none;
border-radius: 10px;
background-color: #6d747c;
color: #fff;
}

:hover {
background-color: #7c8289;
}</string>
           </property>
           <property name="text">
            <string>Limpar campos</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="3">
        <widget class="QGroupBox" name="dadosProduto">
         <property name="title">
          <string>Dados do produto</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Código do produto:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="codProd">
              <property name="styleSheet">
               <string notr="true">color: #bfbebd;</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Descrição do produto:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="descricao">
              <property name="maxLength">
               <number>40</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Quantidade:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="quantidade">
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2" rowspan="3">
        <widget class="QGroupBox" name="precoProduto">
         <property name="title">
          <string>Precificação</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_5">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Valor de compra:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="valorCompra">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="prefix">
               <string>R$ </string>
              </property>
              <property name="maximum">
               <double>999999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Valor de venda:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="valorVenda">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">padding-right: 10px;</string>
              </property>
              <property name="showGroupSeparator" stdset="0">
               <bool>false</bool>
              </property>
              <property name="prefix">
               <string>R$ </string>
              </property>
              <property name="maximum">
               <double>999999.000000000000000</double>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Recursos.qrc"/>
 </resources>
 <connections/>
</ui>
